react/ConditionLayoutCategory.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react'
+import type { ComponentType } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
+  conditions: Array<Condition<ContextValues, HandlerArguments>>
+  matchType?: MatchType
+  Else?: ComponentType
+  Then?: ComponentType
+}
+
+type ContextValues = {
+  id: string,
+  type: 'category' | 'department'
+}
+
+type HandlerArguments = {
+  categoryId: { ids: string[] }
+  departmentId: { ids: string[] }
+}
+
+const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
+  categoryId({ values, args }) {
+    if(values.type === 'category') {
+
+      return args.ids.includes(values.id)
+    }
+
+    return false; 
+  },
+  departmentId({ values, args }) {
+    if(values.type === 'department') {
+      
+      return args.ids.includes(values.id)
+    }
+
+    return false;
```

Comment:
Also here, is the same suggestion that I left above.
```suggestion
    if(values.type !== 'department') {
      return false
    }

    return args.ids.includes(values.id)
```